symfony/framework-bundle/4.2/src/bootstrap.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+require __DIR__.'/../vendor/autoload.php';
+
+use Symfony\Component\Dotenv\Dotenv;
+
+if (class_exists(Dotenv::class)) {
+    (new Dotenv())->loadForEnv($_SERVER['APP_ENV'] ?? $_ENV['APP_ENV'] ?? 'dev', __DIR__.'/../.env');
```

Comment:
this change does not allow using `.env` to specify the value for `APP_ENV`, while it is the current documented way to run the app in a different environment easily locally today.

AFAICt, changing the env locally now requires using a real env variable.

Reply:
You are referring to https://symfony.com/doc/current/configuration/environments.html#executing-an-application-in-different-environments, right?

AFAIU, this still works (`$env` still is obtained in front controllers by executing: 
```php
$env = $_SERVER['APP_ENV'] ?? $_ENV['APP_ENV'] ?? 'dev'
```
after loading the `.env` file). 

But for convenience, if you really need to access often your app in another env but cannot provide it in your server configuration, I'd rather recommend creating a dedicated front controller. Not to be commit/deployed of course.

Just changing the `APP_ENV` value in `.env` will change the environment used by the app, but will keep all other env vars dev values, which seems a bit weird.

Reply:
> this change does not allow using .env to specify the value for APP_ENV, while it is the current documented way to run the app in a different environment easily locally today.

This to me is a no-go as it has been teached like this since introducing the .env in sf. Breaking this behavior would catch alot of people offguard.

Reply:
As I said above, unless I’m missing something, this still works. But I really don’t think that’s the easiest way.

Reply:
@Tobion this isn't the case, as explainded by @ogizanagi: https://github.com/symfony/recipes/pull/466/files#diff-7d73eabd1e5eb7d969ddf9a7ce94f954R9

Reply:
I updated the PR, can you check again please?